hack/vcluster/partials/main.go

```diff
@@ -93,17 +92,36 @@ var paths = []string{
 }
 
 func main() {
-	_ = os.RemoveAll(OutDir)
-	util.DefaultRequire = false
+	if len(os.Args) != 3 {
+		panic("expected to be called with vcluster jsonschema directory as first argument and output dir as a second, e.g.\n" +
+			"go run hack/vcluster/partials/main.go configsrc/v0.21/ vcluster_versioned_docs/version-0.21.0/_partials/config")
+	}
 
+	util.DefaultRequire = false
+	versionDir := os.Args[1]
+	jsonSchemaPath := versionDir + "/vcluster.schema.json"
+	defaultValues := versionDir + "/default_values.yaml"
+	values, err := os.ReadFile(defaultValues)
+	if err != nil {
+		panic(err)
```

Comment:
A more descriptive error message

`panic(fmt.Errorf("failed to read default values from %q: %w", defaultValues, err))`